src/main/resources/mavenPostgreSQL/liquibase.properties

```diff
@@ -0,0 +1,4 @@
+changeLogFile: dbchangelog.xml
+url: jdbc:postgresql://localhost:5432/postgres
+username: user
+password: mysecretpassword
```

Comment:
Na każdym środowisku adres do bazy danych może być inny - tak samo jak hasło i nazwa użytkownika. Te wartości nalezy sprametryzować poprzez zmienne środowiskowe, które każdy developer ustawi sobie potem sam.
podpowiedź: robi się to przez ${NAZWA_ZMIENNEJ}. gdzie NAZWA_ZMIENNEJ ustawia się potem no w intelliJ jesli uruchamiasz aplikację z IDE